mod-data-import-converter-storage-server/src/main/java/org/folio/services/association/CommonProfileAssociationService.java

```diff
@@ -73,10 +87,10 @@
 
     Future<Optional<ProfileSnapshotWrapper>> result = Future.future();
 
-    jobToActionProfile.getDetailProfilesByMasterId(masterId, detailType, query, offset, limit, tenantId)
+    profileAssociationDao.getDetailProfilesByMasterId(masterId, detailType, query, offset, limit, tenantId)
       .setHandler(ar -> {
         if (ar.failed()) {
-          LOGGER.debug("I could not get details profiles by master id %s, for the tenant %s", masterId, tenantId);
+          LOGGER.debug("Could not get details profiles by master id %s, for the tenant %s", masterId, tenantId);
```

Comment:
placeholder in parametrized logging is {} not %s, and the level should be error not debug